templates/tfengine/recipes/resources.hcl

```diff
@@ -166,6 +168,11 @@ schema = {
             description = "Name of the cloud sql instance."
             type        = "string"
           }
+          type = {
```

Comment:
nit: Move type below resource_name. I would like to group resource_name with the field it uses by default if not present (e.g. name).

Reply:
Done